news/2023/2023-01-21-new-featured-artist-kakichoco.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "New Featured Artist: kakichoco"
+date: 2023-01-21 11:00:00 +0000
+---
+
+We're proud to add legendary vocalist **kakichoco** to our collection of Featured Artists!
+
+![](https://assets.ppy.sh/artists/336/header.jpg)
+
+You'd expect the performance points algorithm to be *more* balanced right after a rework, but **kakichoco**'s [Zan'ei](https://osu.ppy.sh/beatmapsets/1759729) proved that wrong.
+
+Ever since [mrekk](https://osu.ppy.sh/users/7562902)'s [1,244pp score](https://osu.ppy.sh/scores/osu/4185178306) last summer (since rebalanced to *only* 1,033pp), **kakichoco** has become a household name in every osu! player's beatmap library, and we're here to pull her into our Featured Artist library too!
```

Comment:
```suggestion
Ever since [mrekk](https://osu.ppy.sh/users/7562902)'s [1,244 pp score](https://osu.ppy.sh/scores/osu/4185178306) last summer (since rebalanced to *only* 1,033 pp), **kakichoco** has become a household name in every osu! player's beatmap library, and we're here to pull her into our Featured Artist library too!
```